### ISSUE TITLE
Does not connect with Github during registration

### ISSUE THREAD

**Issue by swanidhi** [BloopAI/bloop#1049]
![image](https://github.com/BloopAI/bloop/assets/1059929/c77617ec-ba5c-45e4-87f9-6775b30e6a37)
 
Tried installing the 0.5.5 and 0.5.6 versoins but coulnd;t get through registration process. The connection page confirms successful authentication however th wpplication does not refresh to reflect the same, preventing me to complete registration.

Using Macbook Pro M2

Best,
Swan

**Comment by sampatkalyan** [comment]
Same bug

**Comment by ggordonhall** [comment]
Thanks for reporting this. Did you manually copy the authentication link into the browser? If so, can you confirm that the copied link exactly matches what is displayed here? There is a case where incorrect links can still lead to an "authentication successful" screen.

**Comment by swanidhi** [comment]
The link was accurate. Tried using different browsers too, no avail unfortunately.

**Comment by ggordonhall** [comment]
I've had a look at this and found that you can get this behaviour if there is no network connection during authorisation. There's a patch to fix that: https://github.com/BloopAI/bloop/issues/1049.

Can you try deleting your bloop index directory, ensuring that you have internet connection, and running through the setup process again?

**Comment by ggordonhall** [comment]
@nightscape Does it work if you use it outside of your company network?

**Comment by nightscape** [comment]
@ggordonhall yes, without any issues. Outside is on Mac though and inside on Windows.

**Comment by schmkr** [comment]
I'm also facing this issue. Tried 3 different networks (home connection, mobile 5G connection and a VPS hosted at Hetzner). All result in the same error:

```
Could not resolve host: production-github-oauth.auth.eu-west-1.amazoncognito.com
```

**Comment by juanmarin-co** [comment]
Facing the same issue here, Could not resolve host
